### DISCUSSION TITLE
How to make only part of the content scrollable?

### DISCUSSION THREAD

**Discussion by smikheiev** [gorhom/react-native-bottom-sheet#1724] [Q&A]
Hi! I need to make only a part of the content scrollable inside `BottomSheetModal`. For example, if I want to have a custom header on the top and a footer on the bottom, and some scrollable content between them?
Smth like this (pseudocode):
```
<BottomSheetModal ...>
  <Header ... />
  <ScrollableContent ... />
  <Footer ... />
</BottomSheetModal>
```

I tried using `BottomSheetScrollView`, but it makes all content scrollable. I can use `stickyHeaderIndices` prop for the header, but there's nothing for the footer.
I also tried to use `BottomSheetView` and add a `ScrollView` inside, but it didn't work correctly.

Is there a good way to do it?

**Comment by smikheiev** [comment]
I found that there's `footerComponent` prop, but found one issue with it https://github.com/gorhom/react-native-bottom-sheet/issues/1725 🤔

**Comment by yolpsoftware** [comment]
You ever found an answer?

**Reply by smikheiev** [reply]
uhhh, it's been a while ago. IIRC, I didn't manage to make it work the way I planed, but I found another solution. sorry, I don't really remember anymore how I did it 🙈

**Reply by boutterudy** [reply]
Hi guys 👋

I also encountered the problem, and the solution was pretty simple: use `ScrollView` from `react-native-gesture-handler` ✨

You can find that in [the official documentation](https://gorhom.dev/react-native-bottom-sheet/troubleshooting/#adding-horizontal-flatlist-or-scrollview-is-not-working-properly-on-android) 💯
